<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>690</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>200000</width>
    <height>200000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="maximumSize">
       <size>
        <width>721</width>
        <height>571</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="head_sculpture">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>170</y>
         <width>110</width>
         <height>110</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="picture.qrc">:/picture/2021210.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>721</width>
         <height>221</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="up_background">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>721</width>
          <height>221</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>721</width>
          <height>221</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="pixmap">
         <pixmap resource="picture.qrc">:/picture/lucy.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>310</y>
         <width>421</width>
         <height>191</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QComboBox" name="select_ID">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <italic>false</italic>
              <bold>false</bold>
              <underline>false</underline>
              <strikeout>false</strikeout>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">padding-left: 6px</string>
            </property>
            <item>
             <property name="text">
              <string>2021210</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2021211</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2021212</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2021213</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2021214</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2021215</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2021216</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2021217</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2021218</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2021219</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>88888888</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="code_lineedit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">padding-left: 6px</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
            <property name="placeholderText">
             <string>  密码</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QCheckBox" name="auto_logon">
              <property name="text">
               <string>自动登录</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="remeber_code">
              <property name="text">
               <string>记住密码</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="forget_code">
              <property name="text">
               <string>忘记密码</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="log_on">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>仿宋</family>
              <pointsize>12</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="mouseTracking">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(35, 202, 254);</string>
            </property>
            <property name="text">
             <string>登录</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="down_background">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>230</y>
         <width>721</width>
         <height>341</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_wrong">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>520</y>
         <width>271</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <zorder>down_background</zorder>
      <zorder>groupBox</zorder>
      <zorder>gridLayoutWidget</zorder>
      <zorder>head_sculpture</zorder>
      <zorder>label_wrong</zorder>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>962</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="picture.qrc"/>
 </resources>
 <connections/>
</ui>
